amti/actions/review.py

```diff
@@ -28,7 +29,7 @@ def review_hit(
 
     Returns
     -------
-    None.
+    List of marked assignments.
```

Comment:
Modifying this doc string a bit could help developers more quickly grok the shape of the returned object. After the changes suggested at amti/actions/review.py#L98:

```
List[Dict[str, str]]
    A list of dictionaries representing the marked assignments. Each dictionary
    has ``"assignment_id"``, ``"action"``, and ``"reason"`` keys.
```

Reply:
Done